src/FunctionLoader.ts

```diff
@@ -70,6 +71,19 @@ export class FunctionLoader implements IFunctionLoader {
             throw new InternalException(`Function code for '${functionId}' is not loaded and cannot be invoked.`);
         }
     }
+
+    isESModule(filePath: string, packageJson: PackageJson): Boolean {
+        if (filePath.endsWith('.mjs')) {
+            return true;
+        }
+        if (filePath.endsWith('.cjs')) {
+            return false;
+        }
+        if (packageJson['type'] && packageJson['type'] === 'module') {

```

Comment:
nit:

```suggestion
        if (packageJson.type === 'module') {
```

Reply:
I think the method can be simplified to:
`return filePath.endsWith('.mjs') || packageJson.type === 'module';`

Reply:
@alrod That wouldn't honor the case in which we want `.cjs` to override `packageJson.type === 'module'`.